docs/specification.md

```diff
@@ -240,10 +261,54 @@ wqet7IWbw2j2lmGuoKCaFlYS_R7szczz
 ```
 
 
+#### Terminology
 
----
+Because the encoding algorithm is recursive, this leads to a few different ways to represent a sequence collection. We refer to these representations in "levels". The level number represents the number of "lookups" you'd have to do from the "top level" digest. So, we have:
 
-### 2. API: A server RESTful API specification for retrieving and comparing sequence collections.
+##### Level 0 (AKA "top level")
+
+Just a plain digest. This corresponds to **0 database lookups**. Example:
+```
+a6748aa0f6a1e165f871dbed5e54ba62
+```
+
+##### Level 1
+
+What you'd get when you look up the digest with **1 database lookup** (no recursion). We sometimes refer to this as the "array digests" or "attribute digests", because it is made up a digest for each attribute in the sequence collection. Example:
```

Comment:
```suggestion
What you'd get when you look up the digest with **1 database lookup**. We sometimes refer to this as the "array digests" or "attribute digests", because it is made up a digest for each attribute in the sequence collection. Example:
```